code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -208,6 +208,71 @@
 	M.mind.add_antag_datum(newTraitor)
 	return TRUE
 
+//////////////////////////////////////////////
+//                                          //
+//                 FAMILIES                 //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/midround/families
+	name = "Family Head Aspirants"
+	persistent = TRUE
+	antag_flag = ROLE_FAMILIES
+	restricted_roles = list("Cyborg", "AI", "Prisoner","Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Head of Personnel")
+	required_candidates = 2 // will always have at least 1 family + opponent for that family (other family or cop)
+	weight = 1
+	cost = 25
+	requirements = list(101,101,101,90,80,70,30,15,10,10) // more strict than the roundstart ruleset, because it's midround
+	high_population_requirement = 10
+	flags = TRAITOR_RULESET
```

Comment:
```suggestion
```
TRAITOR_RULESET is for classic secret mode and shouldn't be used on a ruleset like this.